index.md

```diff
@@ -30,6 +30,9 @@ There are also a number of tutorials available at [JuMPTutorials.jl](https://git
 If you need help, search the [Forum](https://discourse.julialang.org/c/domain/opt)
 for similar questions, or open a new one.
 
+There is also an excellent book, [Julia Programming for Operations Research](http://www.chkwon.net/julia/), which covers a variety of topics on optimization, with an emphasis on
```

Comment:
"excellent" is maybe too much of an official endorsement. We would also list other future books written about JuMP (assuming they're reasonably good). Perhaps "see also".